.travis.yml

```diff
@@ -0,0 +1,10 @@
+sudo: required
+services:
+  - docker
+script:
+  - hooks/pre_build
+  # Docker Hub hack
+  - sudo cp -av ./Dockerfile.template ./Dockerfile
```

Comment:
I don't understand why the steps on lines 5-8 are required here. The rendering and build should be done correctly by the `make` call on line 8.

Reply:
hooks/pre_build move it to Dockerfile.template https://github.com/container-images/container-image-template/blob/master/hooks/pre_build#L1

Reply:
See https://github.com/container-images/tools/blob/master/.travis.yml#L5